### PR TITLE
feature/GH-9828: add e2e test for order cancellation

### LINKED ISSUE #9828: Add E2E tests for order cancel/return
Add e2e test for order cancellation and return.

### PR DESCRIPTION
**Pull request by WeizhengSap**
close #9828

### CODE REVIEW

File: projects/storefrontlib/src/cms-components/myaccount/order/amend-order/returns/return-order/return-order.component.html [hunk 1]
```diff
@@ -4,6 +4,8 @@
   <cx-amend-order-items *ngIf="entries$ | async as entries" [entries]="entries">
   </cx-amend-order-items>
 
+  <cx-form-errors [control]="form.get('entries')"></cx-form-errors>
```
Comment (bgambocjaviniar) [inline projects/storefrontlib/src/cms-components/myaccount/order/amend-order/returns/return-order/return-order.component.html:7]: would this be considered a breaking changes as we're adding something new to the template

**Review by bgambocjaviniar — CHANGES_REQUESTED** [review]
https://github.com/SAP/spartacus/pull/9835/files#diff-810e9d5479e5730cd85c51ae7b794aba7380d9ee93648adfcddd9ffe9f6535b6R45

https://github.com/SAP/spartacus/pull/9835/files#diff-810e9d5479e5730cd85c51ae7b794aba7380d9ee93648adfcddd9ffe9f6535b6R86

Getting consistent failures from here from above in the `describe of cancel and return`. Reason being is using `findByText` from the 'inner' helper functions. I would avoid using findByText from now on as I have notice it to cause some flaky test, where it works one day, but fails another. I have done this change for other e2e that failed recently due to this specific chainer.

In conclusion, I suggest to do for example, `cy.get('cx-ammend-order-actions #someselector).should('contain', 'someText').click()` in order to verify that the button you're clicking is the one you want, which behave similarly to `findByText` but is 100% supported without any known flakiness.

File: projects/storefrontlib/src/cms-components/myaccount/order/amend-order/returns/return-order/return-order.component.html (hunk 1 above)
Comment (znikola) [reply projects/storefrontlib/src/cms-components/myaccount/order/amend-order/returns/return-order/return-order.component.html:7]: To me, this looks like a bug fix, therefore let's include it in the rc1 release.

**Review by bgambocjaviniar — APPROVED** [review]
